### PR TITLE
Devop/sentry fixes

### PR DESCRIPTION
**Pull request by gamalielhere**
### Devop

- [x] Updated CHANGELOG.md
- [x] Add PR label

### CODE REVIEW

File: src/wallets/hardware/trezor/index.js [hunk 1]
```diff
@@ -38,7 +39,9 @@ class TrezorWallet {
   getAccount(idx) {
     const derivedKey = this.hdKey.derive('m/' + idx);
     const txSigner = async tx => {
-      tx = new Transaction(tx, {
+      const locTx = Object.assign({}, tx);
+      locTx['chainId'] = new BigNumber(tx['chainId']).toNumber();
```
Comment (kvhnuke) [inline src/wallets/hardware/trezor/index.js]: why is this necessary?

Comment (gamalielhere) [reply src/wallets/hardware/trezor/index.js]: Sentry error that expects `chainId` to be an Integer.

Comment (gamalielhere) [reply src/wallets/hardware/trezor/index.js]: Reverted
